patches/tModLoader/Terraria/ModLoader/TileLoader.cs

```diff
@@ -711,10 +710,14 @@ public static bool Convert(int i, int j, int conversionType)
 			return false;
 		}
 
-		if (tileConversionDelegates[type][conversionType] != null) {
-			foreach (var hook in CollectionsMarshal.AsSpan(tileConversionDelegates[type][conversionType])) {
-				if (!hook(i, j, type, conversionType)) {
-					return false;
+		var conversions = tileConversionDelegates[type];
```

Comment:
Turns out you can do `var list = tileConversionDelegates[type]?[conversionType]`